pkg/agent/manager/sync.go

```diff
@@ -136,8 +136,10 @@ func (m *manager) updateSVIDs(ctx context.Context, log logrus.FieldLogger, c SVI
 
 		update, err := m.fetchSVIDs(ctx, csrs)
 		if err != nil {
+			m.csrSizeLimitedBackoff.Failure()
```

Comment:
may we add a defer, and validate if no error returned, and based on that play with backoff?

Reply:
moved it closer to server call